### ISSUE TITLE
PK-Sim Version 9.0 stopped working

### ISSUE THREAD

**Issue by arketkar** [Open-Systems-Pharmacology/Forum#555]
Hi
Today I installed the new version 9 of PK-Sim. After installation and restarting my laptop, I tried to open the program, however it just crashed down immediately with the following message.
![image](https://user-images.githubusercontent.com/32693494/85429016-b5084d80-b59b-11ea-8014-8b2bf41ec6ac.png)
Then I uninstalled and re-installed the entire setup, however I faced the same problem. So I 'repaired' the setup through control panel. Still the same problem....its not opening at all. MoBi is opening though.

I had installed Version 8 on the same system and it was running fine. My system configuration is -
Windows 7 Professional with SP1, 64 bit OS; 4 GB RAM and 1 TB hard disk.

Appreciate if you could help resolve this issue.

Thanks and regards.
Anant

**Comment by Yuri05** [comment]
@arketkar Which language settings does your operating system have?

Also please export error information from you windows event log and attach it here. For this:
* Open Event Viewer (Run → **eventvwr.msc**).
* Right click on the **Windows Logs/Application** node an select `Filter Current Log...`
![grafik](https://user-images.githubusercontent.com/25061876/85435345-9ac47480-b587-11ea-8609-9f7cad240f47.png)
* Select `Critical `and `Error `event levels and confirm with OK
![grafik](https://user-images.githubusercontent.com/25061876/85435606-00b0fc00-b588-11ea-99a4-4d103f1ea6d8.png)
* Right click on the **Windows Logs/Application** node again and select `Save Filtered Log File As...`
![grafik](https://user-images.githubusercontent.com/25061876/85435960-8a60c980-b588-11ea-9fa3-a8a0ba7fc618.png)
* save in the (default) *.evtx format 
* If the prompt below appears: activate "Display information for these languages" and select "English" from the list (if the language list is empty: activate "Show all available languages first"
![grafik](https://user-images.githubusercontent.com/25061876/85436243-00653080-b589-11ea-87f4-c7d4db00660c.png)
* Zip created .evtx file and attach it here

**Comment by arketkar** [comment]
Hi Yuri,

Thanks for your reply. Please find attached the log file. Sorry I do not know the language setting, I guess it would be English. Please let me know how to find that, in case you need that information.

Thanks.
Anant

[Application error log_arketkar.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/4821040/Application.error.log_arketkar.zip)

**Comment by arketkar** [comment]
Hi Yuri,

![Language setting](https://user-images.githubusercontent.com/32693494/85438538-ba6c9480-b5a9-11ea-96d9-a23e06d5bd3c.jpg)
This is what I could find about the language setting. I hope you were looking for the same.

Regards.
Anant

**Comment by Yuri05** [comment]
@arketkar Can you unzip the file ApplicationSettings.xml and copy it to 
C:\ProgramData\Open Systems Pharmacology\PK-Sim\9.0\

(overwrite existing file if required)
[ApplicationSettings.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/4821669/ApplicationSettings.zip)

(the folder is hidden; if it's not shown - just enter the folder name in Windows Explorer and press ENTER)
![grafik](https://user-images.githubusercontent.com/25061876/85457796-b9833500-b5a0-11ea-9c52-99a9ba10c8c7.png)

Can you start PK-Sim now?

**Comment by arketkar** [comment]
Hi @Yuri05 

Thanks for sharing that file.

I followed all the steps recommended by you. The program started, however when I kept it open for some time (without any activity) I got the same error message and it shut down. Then I restarted my laptop and opened an earlier created model (created on PK-Sim version 8) and ran one DDI simulation. The moment I clicked on the 'Export PK Analysis to csv' button, I again got the same error message and it crashed down.
![Error_Export PK Analysis_24062020](https://user-images.githubusercontent.com/32693494/85526310-83d55f00-b627-11ea-8419-6238d9b10d5b.JPG)


[Error log_24062020_arketkar.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/4824264/Error.log_24062020_arketkar.zip)

I am attaching the new error log file for your reference. Appreciate if you could help to resolve this issue.

Thanks and regards.
Anant

**Comment by arketkar** [comment]
Hi @Yuri05 

Is it possible to have more than one versions of PK-Sim active at a time? At present I am stuck, as Version 9 is not running, and I am not able to run version 8, as it was over-written by version 9.

Regards.
Anant

**Comment by PavelBal** [comment]
@arketkar You can download the portable version of PK-Sim https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/tag/v8.0
Just unzip it somewhere.

**Comment by arketkar** [comment]
Hi @PavelBal 
Thanks for sharing that link. Just to clarify, will that run parallel to version 9 on the same system?
Regards.
Anant

**Comment by PavelBal** [comment]
Yes, it will not overwrite anything and you can run both versions in parallel.

**Comment by arketkar** [comment]
Thank you @PavelBal  It is working in-parallel.

However my problem with PK-Sim version 9 is yet to be resolved. I am sure @Yuri05 will come up with some solution. 

Regards.
Anant

**Comment by msevestre** [comment]
@arketkar Going to move this issue to the PKSim repository as it seems to be related to your computer configuration
